INSTALL.md

```diff
@@ -1,25 +1,27 @@
 ## Installation Steps
 
-Helm charts are provided inside https://github.com/eclipse-tractusx/sd-factory
-
-1.) Using helm commands: <br />
+Helm chart is provided inside https://github.com/eclipse-tractusx/sd-factory
```

Comment:
Please make the sentence nicer:
A helm chart is provided inside the [charts](./charts) directory.

Reply:
Updated with appropriate sentence